### DISCUSSION TITLE
rabbitmq has some err log

### DISCUSSION THREAD

**Discussion by springmvcGitHub** [rabbitmq/rabbitmq-server#6854] [Other]
hi rabbitmq expert.
i use rabbitmq docker version 3.11.0, Corresponding  Erlang version 25.1.1.
There are no error logs when docker run on my own computer, but after deploying this image to the k8s on the cloud, there will be these error logs, which are always printed (rabbitmq runs normally when these error logs appear).

[rabbitmq_err_log.log](https://github.com/rabbitmq/rabbitmq-server/files/10389968/rabbitmq_err_log.log)

**Comment by michaelklishin** [comment]
According to this log, MQTT connections [have to terminate with an "undefined" reason](https://github.com/rabbitmq/rabbitmq-server/blob/v3.11.0/deps/rabbitmq_mqtt/src/rabbit_mqtt_reader.erl#L226-L265), which the code does not
expect.

My best guess is that this is due to [high connection churn](https://rabbitmq.com/connections.html#high-connection-churn) and very short-lived connections.
Alternatively it can be an indicator of a failing TLS connection upgrade: connections are so short lived because TLS clients connect to a non-TLS port and immediately run into a parser error
before they can be established.

Since those connections already are about to terminate, this is simply log noise.

**Comment by janiu-001** [comment]
@michaelklishin  I also had the same error, it is rolling on my screen. Would you help to share your suggestions on how to let them gone?

**Reply by michaelklishin** [reply]
Wait for a new 3.11 patch release or use long lived MQTT connections
